docker-compose.yml

```diff
@@ -0,0 +1,8 @@
+  version: '3.9'
```

Comment:
Well :) let me emphasise more precisely:

Consider (.) dots as space:


version: '3.9'
services:
..web:
....build:
......context: .
......dockerfile: Dockerfile
....ports:
......- "8000:8000"


As an easy way, please copy this code block:

```yaml
version: '3.9'
services:
  web:
    build:
      context: .
      dockerfile: Dockerfile
    ports:
      - "8000:8000"
```

and then paste into docker-compose.yml file.


PS: From the beginning, what I mean is:

```diff
-version: '3.9'
-services:
-    web:
-        build:
-            context: .
-            dockerfile: Dockerfile
-        ports:
-            - "8000:8000"
+version: '3.9'
+services:
+  web:
+    build:
+      context: .
+      dockerfile: Dockerfile
+    ports:
+      - "8000:8000"
```

Reply:
I should have probably said "2-space indentation" instead of "2 spaces" :)
Thanks for your all efforts 🙏🏼

Reply:
@sadikkuzu got it, i have updated please check , it must be fine now